e2e/smoke_test/smokeTest.spec.js

```diff
@@ -76,75 +26,115 @@ describe('smoke test', () => {
             await expect(match.accessible.Label('Glucose')).toExist();
         });
     });
-    describe('cgm', () => {
-        it('can be added', async () => {
-            await cgm.AddSimulator();
-        });
-        it('can configure simulator model', async () => {
-            await cgm.Model(cgm.Models.Constant, ['114']);
-        });
-        it('can configure simulator effect', async () => {
-            await cgm.Effect(cgm.Effects.GlucoseNoise);
+    describe('settings', () => {
+        beforeAll(async () => {
+            await settings.Open();
         });
-        it('can remove data', async () => {
-            await cgm.RemoveData();
+        afterAll(async () => {
+            await settings.Close();
         });
-        it('can be removed', async () => {
-            await cgm.Remove();
+
+        describe('general', () => {
+            it('can be go set to closed loop', async () => {
+                await settings.ClosedLoop();
+            });
+            it('can be go set to open loop', async () => {
+                await settings.OpenLoop();
+            });
+            it('can issue a report', async () => {
+                await settings.IssueReport();
+            });
         });
-    });
-    describe('pump', () => {
-        it('can be added', async () => {
-            await pump.AddSimulator();
+        describe('cgm', () => {
+            it('can be added', async () => {
+                await settings.AddCGMSimulator();
+            });
+            it('can configure simulator model', async () => {
+                await settings.SetCGMModel(CGMModel.Constant, ['114']);
```

Comment:
[nit] Consider handling simulator settings much like you do loop settings. After you add the simulator, then a command to go into the simulator settings, and a way to set each setting. Not needed for now, but if you find you are modifying the CGM/pump simulator settings frequently.